packages/xo-server/bin/xo-server

```diff
@@ -18,7 +18,13 @@ try {
 
 // Use Bluebird for all promises as it provides better performance and
 // less memory usage.
-global.Promise = require('bluebird')
+const Bluebird = require('bluebird')
+
+// Enable `async_hooks` because it's used by `node-zone`
+// See: http://bluebirdjs.com/docs/api/promise.config.html#async-hooks
```

Comment:
```suggestion
// Enable `async_hooks` because it's used by `@xen-orchestra/backups/Task` via `node-zone`
//
// See: http://bluebirdjs.com/docs/api/promise.config.html#async-hooks
```